### DISCUSSION TITLE
@aptos-labs/ts-sdk

### DISCUSSION THREAD

**Discussion by rudreshkoranne** [aptos-labs/aptos-developer-discussions#553] [Questions]
### Describe your question in detail.

When I'm trying to install the ts-sdk using 
`npm i @aptos-labs/ts-sdk` it showing me error as 
`npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/@aptos-labs%2fts-sdk - Not found
npm error 404
npm error 404  '@aptos-labs/ts-sdk@^1.33.1' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.
npm error A complete log of this run can be found in: /home/hp/.npm/_logs/2024-12-08T18_51_26_804Z-debug-0.log
![Screenshot from 2024-12-09 00-18-02](https://github.com/user-attachments/assets/3f377f58-c87b-4165-b7b6-6c74cd748841)

`

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by Pcharlesme** [comment]
@rudreshkoranne Check this out https://www.npmjs.com/package/@aptos-labs/ts-sdk/v/1.9.1-tmp.2

**Comment by Pcharlesme** [comment]
Sometimes, NPM caching can cause errors. Clear the cache and try again:
npm cache clean --force also Package aptos is no longer supported,
you may add  sudo  as prefix if required
sudo npm i @aptos-labs/ts-sdk

**Reply by rudreshkoranne** [reply]
Thank you very much for the help sir!!!
